code/modules/Economy/AdminTools.dm

```diff
@@ -0,0 +1,109 @@
+/datum/admins/proc/EconomyPanel(action, hrefs)
+    if(!check_rights(R_FUN))	return
+
+    var/dat= "<head><title>Economy Tools</title></head>"
+    var/datum/money_account/detailed_account_view
+    var/creating_new_account
+
+    if(hrefs["econ_panel"] != "open")
+        switch(hrefs["econ_panel"])
+            if("create_account")
+                creating_new_account = 1
+            if("finalise_create_account")
+                var/account_name = hrefs["holder_name"]
+                var/starting_funds = max(text2num(hrefs["starting_funds"]), 0)
+                if ((station_account.money - starting_funds) > 0)
```

Comment:
This means you can't create an account with no starting funds if the station account is empty.